src/plugins/header/options/option/option.tsx

```diff
@@ -22,17 +24,16 @@ export class UIPOption extends ESLBaseElement {
 
   /** {@link UIPRoot's} attribute which is changed by option */
   @attr() public attribute: string;
-  public _active: boolean;
 
   /** Closest playground {@link UIPRoot} element */
   protected $root: UIPRoot;
   protected config: OptionConfig;
 
   /** Builds option element from {@link OptionConfig} */
-  static createEl(optionConfig: OptionConfig): UIPOption {
+  static createEl(optionConfig: OptionConfig) {
     const option = document.createElement('uip-option') as UIPOption;
     option.setAttribute('attribute', optionConfig.optionValue);
-    option.append(optionConfig.svg);
+    option.append(optionConfig.svg.cloneNode(true));
```

Comment:
The same as above: do we really need to use `.cloneNode` here?

Reply:
Answered above